src/data/cassandra/Schema.hpp

```diff
@@ -654,6 +746,10 @@ class Schema {
             );
         }();
 
+        /*
```

Comment:
This creates dead uncompiled code. Let's move it to method, and call it from here, but commented.